pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -688,16 +688,19 @@ def create_model(df_sets, df_parameters, default={}):
     model.p_chi_OutsideCompletionsPad = Param(
         model.s_CP,
         initialize=model.df_parameters["CompletionsPadOutsideSystem"],
+        mutable=False,  # this Param must be immutable since it is used in if statements
```

Comment:
Minor: do we need to keep this in the options? I am guessing we are not specifying the mutable option for all the parameters.

Reply:
The reason I explicitly included `mutable=False` here is more because it gave me a good place to include the comment which explains why these Params have to be immutable.

Reply:
I ended up just removing the whole line.